example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -204,6 +204,7 @@
 			files = (
 			);
 			inputPaths = (
+				"${TARGET_BUILD_DIR}/${INFOPLIST_PATH}",
```

Comment:
not sure why this was added. is that ok?

Reply:
pulled down local and it ran with no problem. I'm not really sure what this is, but seems harmless.